cypress/integration/proposal_administration.ts

```diff
@@ -475,7 +475,9 @@ context('Proposal administration tests', () => {
       cy.contains('Title')
         .parent()
         .find('[data-testid="mtableheader-sortlabel"]')
-        .dblclick();
+        .click();
```

Comment:
Double click to quickly reverse the order cannot work on the user officer table, as the sort is now performed in the backend, meaning the table reloads after the first click.